resolwe_bio/tests/processes/test_reads_filtering.py

```diff
@@ -702,3 +702,40 @@ def test_trimgalore_paired(self):
             ["trimgalore/outputs/mate2_trimmed.fastq.gz"],
             compression="gzip",
         )
+
+    @tag_process("gatk-split-ncigar")
+    def test_split_Ncigar_reads(self):
+        input_folder = Path("splitNcigar_reads") / "input"
+        output_folder = Path("splitNcigar_reads") / "output"
+        with self.preparation_stage():
+            species = "Homo sapiens"
+            build = "GRCh38"
+
+            bam = self.prepare_bam(
+                fn=(input_folder / "chr1_500.bam"),
+                species=species,
+                build=build,
+            )
+            ref_seq = self.prepare_ref_seq(
+                fn=(input_folder / "chr1_1-15000.fasta.gz"),
+                species=species,
+                build=build,
+            )
+
+        splitNcigar = self.run_process(
+            "gatk-split-ncigar",
+            {
+                "bam": bam.id,
+                "ref_seq": ref_seq.id,
+            },
+        )
+
+        self.assertFileExists(splitNcigar, "bam")
```

Comment:
Given that all the outputs are required, checking for output existence is probably not necessary as this is done on the level of resolwe.

Reply:
I added this so that it is obvious what the main output is just from the test.